CHANGELOG.md

```diff
@@ -3,6 +3,58 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 5.5.0
+
+_Feb 3, 2022_
+
+A big thanks to the 10 contributors who made this release possible. Here are some highlights ✨:
+
+- 🎁 Add support for [master/detail](https://mui.com/components/data-grid/group-pivot/#master-detail) (#3387) @m4theushw
+
+  <img src="https://user-images.githubusercontent.com/42154031/152379354-47120aac-2b37-4a90-b311-64b4522283b9.gif" width="814">
+
+- 🌍 Add Danish (daDK) locale (#3800) @kasperfilstrup
+- 📚 Documentation improvements
+- 🐞 Bugfixes
+
+### `@mui/x-data-grid@v5.5.0` / `@mui/x-data-grid-pro@v5.5.0`
+
+#### Changes
+
+- [DataGrid] Add methods to import and export the state (#3593) @flaviendelangle
+- [DataGrid] Fix <kbd>PageUp</kbd> jumping directly to the column header skipping the first row (#3761) @cherniavskii
```

Comment:
```suggestion
- [DataGrid] Fix `<kbd>PageUp</kbd>` jumping directly to the column header skipping the first row (#3761) @cherniavskii
```

Reply:
![image](https://user-images.githubusercontent.com/3309670/152385161-12557c52-af36-44b1-8ee7-f161824d884e.png)


The others keys don't have it :angel

Reply:
The backticks override the kbd formatting: `<kbd>PageUp</kbd>` vs <kbd>PageUp</kbd>.